File: src/test/java/org/zbus/examples/broker/BrokerExample.java
package org.zbus.examples.broker;

import org.zbus.broker.Broker;
import org.zbus.broker.ZbusBroker;

public class BrokerExample {

	public static void main(String[] args) throws Exception {  
		Broker broker = new ZbusBroker(); 
		broker.close();
		
		broker = new ZbusBroker("jvm");
		broker.close();
	} 
	
}

File: clients/DotNET/Zbus.Examples/Mq/BrokerExample.cs
using System;
using System.Threading.Tasks;
using Zbus.Mq;

namespace Zbus.Examples
{
    class BrokerExample
    {
        static async Task Test()
        {
            Broker broker = new Broker();
            await broker.AddServerAsync("localhost:15555");
        }
        static void Main(string[] args)
        {
            Test().Wait();
            Console.ReadKey();
        }
    }
}
